tests/author/test_job.py

```diff
@@ -176,3 +176,69 @@ def test_stop_traversal(self):
         job = Job(root_task, arguments)
         self.assertEqual(len(job.flat_hierarchy["tasks"]), 1)
         self.assertTrue("Elements" not in " ".join([_.title for _ in job.flat_hierarchy["tasks"]]))
+
+    def test_modify(self):
+
+        root_task, arguments = tasks.TASKS_DICT.keys()[0], {"uno": [1, 2, 3], "dos": 2, "tres": 3}
+
+        for task in Plugins().tasks.values():
+            task.flags = task.Flags.PER_ELEMENT
+            task.cmd = lambda x: ["/bin/echo", "Hello World"]
+            task.tags = ["foo", "bar"]
+
+        job = Job(root_task, arguments)
+
+        def _get_attribute_listed(job, type, attribute):
```

Comment:
We should probably also call it `scope` instead of `type` to be consistent with the way it's named in the `Plugins()` singleton.

Reply:
Sounds good